lib/msal-angular/src/msal.guard.ts

```diff
@@ -194,7 +194,8 @@ export class MsalGuard implements CanActivate, CanActivateChild, CanLoad {
     }
 
     includesCode(path: string): boolean {
-        return path.lastIndexOf("/code") === path.length - "/code".length || // path.endsWith("/code")
+        return (path.lastIndexOf("/code") > -1 &&
+            path.lastIndexOf("/code") === path.length - "/code".length) || // path.endsWith("/code")
```

Comment:
Why are we not just using endsWith() here?

Reply:
`String.endsWith` is not supported in IE, unfortunately. Dropping IE support would be a breaking change, so we can update this in MSAL Angular v3.